### DISCUSSION TITLE
Windows config file?

### DISCUSSION THREAD

**Discussion by sausje** [chrisbenincasa/tunarr#496] [General]
I accidentally messed up the install by putting in a wrong backend URL and now I can't change it anymore. I've tried to delete the whole folder but after then trying to setup a fresh install it still somehow migrates.. It's not being saved in appdata folders, not in programdata folders. So where is that config that has the backend URL setting in it??

**Comment by chrisbenincasa** [comment]
This is saved in your browsers local storage. If you clear site data for whatever host you access the Tunarr front end on, it should default back on a fresh reload

**Comment by sausje** [comment]
Awesome thanks. 

I messed up because I'm trying to access the webui (and thus m3u) from a lan IP but also having issues with that. Do I need to make a separate topic for that question?

**Reply by chrisbenincasa** [reply]
Hmm if it's not too much trouble, might be a good idea to make a fresh topic so it's easy to discover if others have the same issue
